exporter/sematextexporter/config.go

```diff
@@ -3,4 +3,69 @@
 
 package sematextexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sematextexporter"
 
-type Config struct{}
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/config/configretry"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+const (
+	euRegion          = "eu"
+	usRegion          = "us"
+	euMetricsEndpoint = "https://spm-receiver.eu.sematext.com"
+	usMetricsEndpoint = "https://spm-receiver.sematext.com"
+)
+
+type Config struct {
+	confighttp.ClientConfig   `mapstructure:",squash"`
+	configretry.BackOffConfig `mapstructure:"retry_on_failure"`
+	// Region specifies the Sematext region the user is operating in
+	// Options:
+	// - EU
+	// - US
+	Region string `mapstructure:"region"`
+	// MetricsConfig defines the configuration specific to metrics
+	MetricsConfig `mapstructure:"metrics"`
+}
+type MetricsConfig struct {
+	// App token is the token of Sematext Monitoring App to which you want to send the metrics.
+	AppToken string `mapstructure:"app_token"`
+	// MetricsEndpoint specifies the endpoint for receiving metrics in Sematext
+	MetricsEndpoint string                     `mapstructure:"metrics_endpoint"`
```

Comment:
This is not described in README. Should it be exposed to the user at all? Looks like any value set by the user will be overwritten anyway in the Validate method https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37642/files#diff-1a997d3d49f6b871479f084e0bf95537fbe6844dc9b0a373f12780e046e3a07aR58-R63.
```suggestion
	MetricsEndpoint string                     `mapstructure:"-"`
```